src/core/RecyclerListView.tsx

```diff
@@ -181,7 +188,7 @@ export default class RecyclerListView<P extends RecyclerListViewProps, S extends
             this._pendingScrollToOffset = offset;
         }, (index) => {
             return this.props.dataProvider.getStableId(index);
-        }, !props.disableRecycling);
+        }, !props.disableRecycling, props.viewabilityConfig);
```

Comment:
viewabilityConfig is just initialized in the constructor any changes to viewabilityConfig after the first render will not be respected. 
This use case would come for pagination later with multiple impression configs at the widget level.